### ISSUE TITLE
Error in comfy ui workflow displaying: blocks.1.pos_embedding.embedding.weight

### ISSUE THREAD

**Issue by Shinotabi** [ChenDarYen/Normalized-Attention-Guidance#13]
Hi there, new to the game and ive been trying to fix this for hours now. When running the workflow it always stops when trying to load the embedding with the following error code: blocks.1.pos_embedding.embedding.weight
Maybe there is an easy fix or im just overloooking the obvious here.

Thx in advance

**Comment by onestardao** [comment]
Looks not like a UI bug. sounds like model weights and network config disagree — positional embedding key missing or renamed.

quick checks:

1. confirm checkpoint matches model config (same version and architecture).
2. run in Python:

```py
for k in model.state_dict().keys():
    print(k)
```

look for `blocks.1.pos_embedding.embedding.weight` (or similar).
3\) if the key is missing, either load the correct checkpoint, or apply a key mapping to rename checkpoint keys to match the model, or re-export the correct weights.

this matches ProblemMap No.5 (Semantic ≠ Embedding).
want the minimal checklist + common fixes (small script for key-mapping)? I can paste it here.
